application/core/LsDefaultDataSets.php

```diff
@@ -2626,7 +2626,7 @@ public static function getBaseQuestionThemeEntries()
                 "name" => "browserdetect",
                 "visible" => "Y",
                 "xml_path" => "themes/question/browserdetect/survey/questions/answer/shortfreetext",
-                "image_path" => "/assets/images/screenshots/S.png",
+                "image_path" => "/assets/images/screenshots/BROWSER.png",
```

Comment:
Maybe we can set the filenames in lower case as to follow other filenames?

Reply:
S are not in lowercase ;) EQUATION or PIPE neither